client/verta/verta/registry/check_model_dependencies.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+from typing import Set, Type
+import warnings
+
+from verta.registry import VertaModelBase
+from verta.environment import Python
+from verta._internal_utils import model_dependencies as md
+
+
+def check_model_dependencies(
+        model: Type[VertaModelBase],
+        environment: Python,
+        raise_for_missing: bool = False,
+        ) -> bool:
+    """
+    Attempt to scan the provided model class for dependencies and verify that
+    the provided environment includes the necessary packages to support them.
+
+    .. version_added:: 0.22.1
+
+    Parameters
+    ----------
+    model: Type[VertaModelBase]
+        Model class, inherited from :class:`~verta.registry.VertaModelBase` to be scanned.
+    environment:
+        Instance of `~verta.environment.Python` to validate against.
+    raise_for_missing: bool, default False
```

Comment:
To be a bit more consistent with [`create_standard_model()`](https://github.com/VertaAI/modeldb/blob/5cf73b6f9d6cd648a7fc180c13540029040f8b76/client/verta/verta/registry/entities/_model.py#L363-L366)

```suggestion
    model_cls : subclass of :class:`~verta.registry.VertaModelBase`
        Model class to be scanned.
    environment : :class:`~verta.environment.Python`
        pip dependencies to validate against.
    raise_for_missing : bool, default False
```

Reply:
Fixed